homeassistant/components/geonetnz_quakes/strings.json

```
@@ -6,6 +6,6 @@
         "data": { "radius": "Radius", "mmi": "MMI" }
       }
     },
-    "abort": { "already_configured": "Location is already configured." }
+    "abort": { "already_configured": "[%key:common::config_flow::abort::already_configured_device%]" }
```

I would not categorize this as a device rather as a service.
```suggestion
    "abort": { "already_configured": "[%key:common::config_flow::abort::already_configured_service%]" }
```

---

thanks for the input. 👍